language/de/ucp.php

```diff
@@ -73,22 +73,26 @@
 		<li>Die Änderungen gelten als anerkannt und verbindlich, wenn der Nutzer den Änderungen zugestimmt hat.</li></ol>
 
 		<p>Informationen über den Umgang mit deinen persönlichen Daten sind in der Datenschutzerklärung enthalten.',
-	'PRIVACY_POLICY'		=> 'Diese Richtlinie beschreibt, wie „%1$s“ („%2$s“) und ggf. verbundene Institutionen (im Folgenden „das Board“) und phpBB die Daten verwenden, die während deines Foren-Besuchs gesammelt werden.</p>
+	'PRIVACY_POLICY'		=> 'Diese Richtlinie beschreibt, wie „%1$s“ („%2$s“) (im Folgenden „der Betreiber“) die Daten verwenden, die während deines Foren-Besuchs gesammelt werden.</p>
 		<h3>Umfang und Art der Datenspeicherung</h3>
-		<p>Deine Daten werden auf zwei verschiedene Arten gesammelt:</p>
-		<ol style="list-style-type: decimal;"><li>phpBB erstellt bei deinem Besuch des Boards mehrere Cookies. Cookies sind kleine Textdateien, die dein Browser als temporäre Dateien ablegt. Zwei dieser Cookies enthalten eine eindeutige Benutzer-Nummer (Benutzer-ID) sowie eine anonyme Sitzungs-Nummer (Session-ID), die dir von phpBB automatisch zugewiesen wird. Ein drittes Cookie wird erstellt, sobald du Themen besucht hast und wird dazu verwendet, Informationen über die von dir gelesenen Beiträge zu speichern, um die ungelesenen Beiträge markieren zu können.</li>
-		<li>Weitere Daten werden gesammelt, wenn Informationen an den Betreiber übermittelt werden. Dies betrifft — ohne Anspruch auf Vollständigkeit — zum Beispiel Beiträge, die als Gast erstellt werden, Daten, die im Rahmen der Registrierung erfasst werden und die von dir nach deiner Registrierung erstellten Nachrichten. Dein Benutzerkonto besteht mindestens aus einem eindeutigen Benutzernamen, einem Passwort zur Anmeldung mit diesem Konto und einer persönlichen und gültigen E-Mail-Adresse.</li></ol>
+		<p>Deine Daten werden auf vier verschiedene Arten gesammelt:</p>
+		<ol style="list-style-type: decimal;"><li>Die Forensoftware phpBB erstellt bei deinem Besuch des Boards mehrere Cookies. Cookies sind kleine Textdateien, die dein Browser als temporäre Dateien ablegt und die zwischen den einzelnen Aufrufen des Boards erhalten bleiben. In diesen Cookies sind die aktuelle ID deiner Sitzung (damit dir alle Seitenaufrufe zugeordnet werden können), Informationen über die von dir gelesenen Beiträge (zur Markierung dieser als gelesen/ungelesen; sofern du nicht angemeldet bist) sowie Informationen über deine Teilnahme an Umfragen (sofern du nicht angemeldet bist) gespeichert. Ferner werden deine Benutzer-ID, ein Authentifizierungsschlüssel und eine Session-ID gespeichert, sofern du beim Anmelden „Angemeldet bleiben“ ausgewählt hast. Die Cookies haben standardmäßig eine Gültigkeit von einem Jahr. Alle Cookies kannst du jederzeit über die Funktion „Alle Cookies löschen“ löschen.</li>
```

Comment:
```suggestion
		<ol style="list-style-type: decimal;"><li>Die Forensoftware phpBB erstellt bei deinem Besuch des Boards mehrere Cookies. Cookies sind kleine Textdateien, die dein Browser als temporäre Dateien ablegt und die zwischen den einzelnen Aufrufen des Boards erhalten bleiben. In diesen Cookies sind die aktuelle ID deiner Sitzung (damit dir alle Seitenaufrufe zugeordnet werden können), Informationen über die von dir gelesenen Beiträge (zur Markierung dieser als gelesen/ungelesen; sofern du nicht angemeldet bist) sowie Informationen über deine Teilnahme an Umfragen (sofern du nicht angemeldet bist) gespeichert. Ferner werden deine Benutzer-ID, ein Authentifizierungsschlüssel und eine Session-ID gespeichert. Die Cookies haben standardmäßig eine Gültigkeit von einem Jahr. Alle Cookies kannst du jederzeit über die Funktion „Alle Cookies löschen“ löschen.</li>
```
Cookies werden doch auch gesetzt, wenn man nicht "angemeldet bleiben" auswählt. Auch für die kurzzeitige Anmeldung werden Cookies verwendet, allerdings geht die Session dann wieder verloren.